src/render/mod.rs

```diff
@@ -799,8 +794,7 @@ impl TextureManager {
         let mut tm = TextureManager {
             textures: HashMap::with_hasher(BuildHasherDefault::default()),
             version: {
-                let ver = res.read().unwrap().version();
-                ver
+                res.read().unwrap().version()
```

Comment:
This broke the build https://travis-ci.org/Thinkofname/steven/jobs/155982067#L902